backend/functions/lib/adapters/gateways/cosmos/cosmos.repository.test.ts

```diff
@@ -4,9 +4,9 @@ import { ApplicationContext } from '../../types/basic';
 import { getCosmosDbCrudRepository } from '../../../factory';
 import { AggregateAuthenticationError } from '@azure/identity';
 import { ServerConfigError } from '../../../common-errors/server-config-error';
-import { CosmosDbRepository } from './cosmos.repository';
 import { ErrorResponse } from '@azure/cosmos';
 import { ID_ALREADY_EXISTS } from './cosmos.helper';
+import { DocumentRepository } from '../../../use-cases/gateways.types';
```

Comment:
What was the reason for switching to DocumentRepository instead of CosmosDbRepository?